diff -up workrave-6f9bc5d9d66c6042923ca39367f54db39ecd914a/configure.ac.fix workrave-6f9bc5d9d66c6042923ca39367f54db39ecd914a/configure.ac
--- workrave-6f9bc5d9d66c6042923ca39367f54db39ecd914a/configure.ac.fix	2013-01-07 12:13:06.000000000 -0500
+++ workrave-6f9bc5d9d66c6042923ca39367f54db39ecd914a/configure.ac	2013-01-08 10:05:16.474764630 -0500
@@ -23,7 +23,7 @@ AC_CONFIG_SRCDIR([backend/include/ICore.
 AC_CONFIG_MACRO_DIR([m4])
 AC_CONFIG_HEADER([config.h])
 
-AM_INIT_AUTOMAKE([-Wall -Werror foreign silent-rules])
+AM_INIT_AUTOMAKE([-Wall -Wno-portability])
 dnl AM_SILENT_RULES([yes])
 
 GNOME_SHELL_VERSION=3.6.2
