diff -up workrave-1.9.2/frontend/gtkmm/src/gnome_applet/Makefile.in.compile workrave-1.9.2/frontend/gtkmm/src/gnome_applet/Makefile.in
--- workrave-1.9.2/frontend/gtkmm/src/gnome_applet/Makefile.in.compile	2010-11-03 12:52:59.000000000 +0100
+++ workrave-1.9.2/frontend/gtkmm/src/gnome_applet/Makefile.in	2010-08-28 14:58:33.000000000 +0200
@@ -496,7 +496,7 @@ SUBDIRS = 
 @HAVE_GNOMEAPPLET_TRUE@	 		 	  -DGNOMELOCALEDIR="\"$(datadir)/locale\"" 
 
 @HAVE_GNOMEAPPLET_TRUE@workrave_applet_SOURCES = WorkraveApplet.c
-@HAVE_GNOMEAPPLET_TRUE@workrave_applet_LDADD = @GNOME_LIBS@
+@HAVE_GNOMEAPPLET_TRUE@workrave_applet_LDADD = @GNOME_LIBS@ @DBUS_LIBS@
 @HAVE_GNOMEAPPLET_TRUE@workrave_applet_CXXFLAGS = ${FLAGS}
 @HAVE_GNOMEAPPLET_TRUE@workrave_applet_CFLAGS = ${FLAGS}
 @HAVE_GNOMEAPPLET_TRUE@BUILT_SOURCES = applet-server-bindings.h applet-client-bindings.h gui-client-bindings.h  Workrave-Applet.server Workrave-Applet.server.in
