--- workrave-1.8.1/configure.ac.mo-ext	2005-10-22 22:33:45.000000000 +0200
+++ workrave-1.8.1/configure.ac	2005-10-22 22:33:54.000000000 +0200
@@ -747,7 +747,7 @@
 
 dnl somehow, these don't get defined anymore...                                  
 CATOBJEXT=.gmo 
-INSTOBJEXT=.po 
+INSTOBJEXT=.mo 
 AC_SUBST(CATOBJEXT)
 AC_SUBST(INSTOBJEXT)
 
--- workrave-1.8.1/configure.mo-ext	2005-07-31 11:24:06.000000000 +0200
+++ workrave-1.8.1/configure	2005-10-22 22:34:08.000000000 +0200
@@ -38643,7 +38643,7 @@
 
 
 CATOBJEXT=.gmo
-INSTOBJEXT=.po
+INSTOBJEXT=.mo
 
 
 
