diff -up workrave-1.8.5/frontend/gtkmm/src/WindowHints.cc.lock workrave-1.8.5/frontend/gtkmm/src/WindowHints.cc
--- workrave-1.8.5/frontend/gtkmm/src/WindowHints.cc.lock	2007-09-14 20:50:25.000000000 +0200
+++ workrave-1.8.5/frontend/gtkmm/src/WindowHints.cc	2008-04-04 10:53:28.000000000 +0200
@@ -185,6 +185,11 @@ WindowHints::grab(int num_windows, GdkWi
           // So, let's not waste memory and simply return a bogus non-NULL ptr.
           handle = (WindowHints::Grab *) 0xdeadf00d;
         }
+      else
+        {
+	  gdk_keyboard_ungrab(GDK_CURRENT_TIME);
+	  gdk_pointer_ungrab(GDK_CURRENT_TIME);
+        }
     }
 #elif defined(WIN32)
   if (num_windows > 0)
